packages/connect-web-test/src/nodeonly/grpc-transport.ts

```diff
@@ -260,7 +260,11 @@ export function createGrpcTransport(
               method,
               header,
               trailer,
-              async read(): Promise<ReadableStreamDefaultReadResult<O>> {
+              // Note this is using the TypeScript type ReadableStreamReadResult and not the custom type used by
+              // Connect-Web.  This mimics how users will interact with the library (using their own DOM lib) and tests
+              // compatibility with the two types in that the Connect Web custom type should always be assignable to
+              // the type in the user's DOM lib.
+              async read(): Promise<ReadableStreamReadResult<O>> {
```

Comment:
Just relying on type inference is probably an easy solution here:

```suggestion
              async read() {
```